content/en/docs/setup/production-environment/index.md

```diff
@@ -0,0 +1,252 @@
+---
+title: "Production environment"
+description: Create a production-quality Kubernetes cluster
+weight: 30
+no_list: true
+---
+<!-- overview -->
+
+A production-quality Kubernetes cluster requires planning and preparation.
+If co-workers and customers depend on Kubernetes being available to run critical
+workloads, it is your job to make sure that the platform is set up and
+maintained to meet their demands.
+
+<!-- body -->
+
+## Production issues
+
+A personal Kubernetes learning or development environment is not meant to be
+used in production. A production environment may require secure access by many
+users, consistent availability, and the resources to adapt to changing demands.
+
+As you decide where you want your production Kubernetes environment to live
+(on premises or in a cloud) and the amount of management you want to take
+on or hand to others, consider how your requirements for a Kubernetes cluster
+are colored by the following issues:
+
+- *Availability*: A single-machine Kubernetes [learning environment](/docs/setup/#learning-environment)
+has a single point of failure. Creating a highly available cluster means considering:
+  - Separating the control plane from the worker nodes.
+  - Duplicating the control plane by having multiple instances of each control plane service available, having those services run on multiple machines, or load balancing traffic to the cluster’s {{< glossary_tooltip term_id="kube-apiserver" text="API server" >}}.
```

Comment:
```suggestion
  - Replicate the control plane components on multiple nodes.
  - Load balance traffic to the cluster’s {{< glossary_tooltip term_id="kube-apiserver" text="API server" >}}.
```
A load balancer for the `kube-apiserver` is required for HA Kubernetes, [reference link](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/high-availability/#first-steps-for-both-methods)

Reply:
Done. I changed to "Replicating" and "Load balancing" to match the other bullets. Otherwise, I used your wording.